docs/docs/sources/chatplugin/overview.md

```diff
@@ -4,13 +4,57 @@ sidebar_label: Overview
 ---
 
 import TLDR from "@site/src/components/TLDR";
+import ButtonBox from "@site/src/components/ButtonBox";
+import ButtonBoxList from "@site/src/components/ButtonBoxList";
+import HammerAndWrenchSVG from "@site/static/icons/hammer-and-wrench.svg";
 
 <TLDR>
 
-Airy's Live Chat Plugin is an **open-source chat widget** that you can put on your website to chat with your visitors.
+Airy's Live Chat Plugin is an open-source chat widget that is fully customizable
+and included in Airy Core
 
  </TLDR>
 
-The Airy Live Chat Plugin is a fully-featured
-[source](/getting-started/glossary#source) that enables conversations with
-anonymous website visitors through a web chat plugin.
+## Introduction
+
+Having a Live Chat plugin on your website has become essential. Connect with
+your website visitors, communicate with them in real time, or use a bot to
+automate FAQs.
+
+Airy’s Live Chat Plugin comes out of the box fully working, but thanks to its
+open-source nature and React Render Props you can customize everything about it.
+
+Out of the box Airy’s Live Chat Plugin supports:
+
+- Full customization of look, feel and features
+- All message types, including emojis
+- Rich Messaging: Templates, cards & carousels
```

Comment:
Templates and cards are the same right?

Reply:
templates in Facebook, cards in Google. better to have all mentioned for people who only know one of the terms